### PR TITLE
Fix getMeta and index messages

### PR DESCRIPTION
**Pull request by osipov-mit**
- Return meta as long as it has `hex` filed that isn't null
- Reindex block with a program not found

### CODE REVIEW

File: idea/indexer/src/common/form-response.middleware.ts
Comment (Almazatun) [inline idea/indexer/src/common/form-response.middleware.ts:1]: Why this file has a prefix with `middleware`? If it's not middleware.

File: idea/indexer/src/services/message.service.ts [hunk 1]
```diff
@@ -71,7 +70,7 @@ export class MessageService {
     };
   }
 
-  public async create(messages: Message[]): Promise<IMessage[]> {
+  public async create(messages: Message[]): Promise<Message[]> {
     for (const m of messages) {
```
Comment (Almazatun) [inline idea/indexer/src/services/message.service.ts:74]: What is the advantage use a single letter? It makes code more readable and what kind of quality improves if using a single letter? In my opinion, it makes more decreases readable quality for other people who will look at this code.

File: idea/indexer/src/common/form-response.middleware.ts
Comment (osipov-mit) [reply idea/indexer/src/common/form-response.middleware.ts:1]: right

File: idea/indexer/src/services/message.service.ts (hunk 1 above)
Comment (osipov-mit) [reply idea/indexer/src/services/message.service.ts:74]: It's good when code is readable and short as much as it possible. In this case it's absolutely clear what `m` means. I don't see any problem

**Review by Almazatun — APPROVED** [review]

**Pull request merged**